helpers.go

```diff
@@ -148,15 +148,15 @@ func (p *Policy) AllowStandardAttributes() {
 	p.AllowAttrs("dir").Matching(Direction).Globally()
 	p.AllowAttrs(
 		"lang",
-	).Matching(regexp.MustCompile(`[a-zA-Z]{2,20}`)).Globally()
+	).Matching(regexp.MustCompile(`^[a-zA-Z]{2,20}$`)).Globally()
```

Comment:
`ˆ` and `$` can be used by attackers. Check if it's possible to use `\A` or `\z`.

Example: `\A[a-zA-Z]{2,20}\z`

Reply:
I think this is fine? Unless the ?m flag is provided, ^ and $ match the beginning and end of the text, not the beginning and end of the line. (See https://pkg.go.dev/regexp/syntax)